prime-router/docs/getting-started/Using-an-apple-silicon-mac.md

```diff
@@ -55,36 +55,28 @@ docker ps
 
 ### Step 4 - Run support services
 
-ReportStream depends on set of services to be up before running main Azure service. These services include:
+ReportStream depends on set of services to be up before running main Azure service. These services include the following 
+and are run automatically when starting ReportStream Dockerless:
 
 * Azurite - a simulator of Azure storage
 * Vault - a secret store
-* SFTP - A SFTP server
+* SFTP - an SFTP server
+* FTPS - an FTPS server
+* soap-webservice - SOAP web service emulator
 
-You can run these services using the `docker-compose` tool. 
-
-```bash
-docker-compose up sftp azurite ftps vault 
-```
-
-Look over the log in your terminal session and check for any errors. 
-If you find any, read the [Things that might go wrong](#things-that-might-go-wrong) section. 
-
-You can take down these services using a "ctrl-c" keyboard combination or the `docker-compose down` command. 
-For now, leave these services running and open up a new terminal session. 
-
-> Note: If you do not want to devote a whole terminal session the logs of these services. 
-> You can run them in detached mode `docker-compose up --detach sftp azurite ftps vault` and then
-> attach to the containers when you want to examine the logs. 
+You can take down these services by running `./gradlew composeDown` or `docker-compose down` command. 
+For now, leave these services running and open up a new terminal session.
 
 ### Step 5 - Run ReportStream locally
-With the dependent services running and a freshly built JAR created by `cleanslate.sh`, we can run ReportStream locally. 
-We use Gradle to launch ReportStream, because Gradle will set up the environment variables that ReportStream needs. 
+Use Gradle to launch ReportStream, as it will set up the environment variables that ReportStream needs. 
 
 ```bash
-gradle quickrun
+./gradlew run
```

Comment:
running this fresh I had to create the `.vault/env/.env.local` file

Reply:
This is not meant to be stand alone.  You have to follow the getting started instructions still.